app/configurator/components/filters.tsx

```diff
@@ -80,6 +80,7 @@ import {
   TemporalDimension,
   TemporalEntityDimension,
 } from "@/domain/data";
+import { isDynamicMaxValue, VISUALIZE_MAX_VALUE } from "@/domain/max-value";
```

Comment:
Suggestion on naming: Maybe we should use VISUALIZE_MOST_RECENT_VALUE ? Seems like MOST_RECENT_VALUE is the sentinel value for most recent value, when max value is the real value from the dimension. What do you think ?

Reply:
Yes, good point, I'll change this 👍